public/static/locales/fr/form-builder-responses.json

```diff
@@ -0,0 +1,97 @@
+{
+  "responses": {
+    "changeSetup": "Modifier les paramètres",
+    "confirmReceipt": "Confirmer la réception",
+    "navLabel": "Réponses",
+    "reportProblems": "Signaler un problème",
+    "title": "Télécharger les réponses",
+    "unauthenticated": {
+      "createOne": "créez-en un",
+      "noAccount": "Si vous n'avez pas de compte Formulaires GC,",
+      "signIn": "connectez-vous",
+      "title": "Se connecter pour accéder aux réponses",
+      "toAccessPagePart1": "Pour accéder à cette page,",
+      "toAccessPagePart2": "à votre compte Formulaires GC"
+    }
+  },
+  "downloadResponsesTable": {
+    "header": {
+      "tableTitle": "Download Responses Table",
+      "select": "Sélection",
+      "number": "Numéro de formulaire",
+      "status": "État des réponses",
+      "downloadResponse": "Délai de téléchargement",
+      "lastDownloadedBy": "Réponses téléchargées par",
+      "confirmReceipt": "Confirmation de réception",
+      "removal": "Suppression des réponses"
+    },
+    "status": {
+      "downloaded":"Téléchargées",
+      "notDownloaded":"Non téléchargées",
+      "new":"Nouveauté",
+      "confirmed":"Confirmées",
+      "unconfirmed": "Non confirmée",
+      "problem":"Problème",
+      "done": "Confirmée",
+      "withinXDays": "Dans {{daysLeft}} jours",
+      "overdue":"En retard",
+      "wontRemove": "Échec de la suppres",
+      "notSet": "Non réglée"
+    },
+    "card": {
+      "noResponses": "Vous n’avez pas de réponses",
+      "noResponsesToDownload": "Il n’y a pas de réponses disponibles pour le téléchargement."
+    },
+    "notifications": {
+      "downloading":"Téléchargement",
+      "downloadComplete": "Téléchargement complet",
+      "downloadingXFiles": "Téléchargement de {{fileCount}} fichiers",
+      "downloadXSelectedResponses": "Télécharger {{size}} réponses sélectionnées",
+      "trySelectingLessFilesHeader": "Seules {{max}} réponses peuvent être téléchargées à la fois.",
+      "trySelectingLessFiles":"Vous avez {{max}} réponses sélectionnées. Décochez au moins 1 réponse avant de télécharger.",
+      "atLeastOneFileHeader": "Au moins un fichier doit être sélectionné à télécharger.",
+      "atLeastOneFile": "Essayez de sélectionner au moins un fichier à télécharger.",
+      "errorDownloadingFilesHeader":"Erreur lors du téléchargement des fichiers sélectionnés.",
+      "errorDownloadingFiles":"Réessayez dans quelques minutes. Si le problème persiste, contactez ",
```

Comment:
Inline links could also be accomplished with the Trans component but I wanted to hold off adding too many new dependencies/ways of doing things for one feature: https://react.i18next.com/latest/trans-component